react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,49 @@
+import './ValidatedInput.css';
+import { useState } from "react";
+
+export default function ValidatedInput () {
+  const [password, setPassword] = useState('');
+  const numReg = /[0-9]+/;
+  const upReg = /[A-Z]+/;
+  const charReg = /[!@#$%^&*()]+/;
+  const passwordLength =
+  (password.length === 0) ?
+    "A password is required."
+: (password.length < 8) ?
+  "Your password is too short."
+:(password.length > 7 && !numReg.test(password)) ?
+"Please include a number"
+: (password.length > 7 && !upReg.test(password)) ?
+ "Please include a capital letter"
+: (password.length > 7 && !charReg.test(password)) ?
+"Please include one of the following characters !, @, #, $, %, ^, &, *, (, or )"
+:
+"";
+
+  return (
+    <>
+    <div className="contrasena">
+      <div>
+        <label htmlFor="password">Password</label>
```

Comment:
Did you try wrapping the `<label>` around the `<input>`? That usually results in a nice layout, and it's a nicer way to use label. Especially since you won't need `htmlFor` in the label or `id` in the input.